CMakeLists.txt

```diff
@@ -440,12 +440,12 @@ if(CMAKE_FORMAT)
                          ${PROJECT_SOURCE_DIR}/scripts/cmake_format_all.sh)
 endif()
 
-if(CMAKE_FORMAT OR CLANG_FORMAT)
-  add_custom_target(format)
-  add_dependencies(format clang-format)
-  if(CMAKE_FORMAT)
-    add_dependencies(format cmake-format)
-  endif()
+# We always define the target 'format' since 'clang-format' is guaranteed to
```

Comment:
What do you mean by 'clang-format is guaranteed to exist'? I'm afraid this is not true. This comment needs a little more context.

Reply:
Oh.... after reading it like 5 times I realized you probably meant the target, not the executable. This is not obvious.